README.md

```diff
@@ -41,14 +41,19 @@ $ kubectl apply -f https://github.com/Dynatrace/dynatrace-operator/releases/late
 A secret holding tokens for authenticating to the Dynatrace cluster needs to be created upfront. Create access tokens of
 type *Dynatrace API* and *Platform as a Service* and use its values in the following commands respectively. For
 assistance please refere
-to [Create user-generated access tokens](https://www.dynatrace.com/support/help/get-started/introduction/why-do-i-need-an-access-token-and-an-environment-id/#create-user-generated-access-tokens)
-.
+to [Create user-generated access tokens.](https://www.dynatrace.com/support/help/get-started/introduction/why-do-i-need-an-access-token-and-an-environment-id/#create-user-generated-access-tokens)
+
+Make sure the *Dynatrace API* token has the following permission:
+
+* Access problem and event feed, metrics and topology
+* Read configuration
+* Write configuration
```

Comment:
Only necessary when deployed via `install.sh`. Would rather provide a separate README next to the script.

Reply:
Removed from here